File: ICT/lab5/g.py
n = int(input())
cnt = 0
while( n>= 25):
    n = n - 25
    cnt +=1

while(n>= 10):
    n = n - 10
    cnt +=1
while(n>= 5):
    n= n- 5
    cnt+=1
cnt+=n

print(cnt)

File: lessons-2/09.12/g.cpp
#include<iostream>

using namespace std;

int main(){
    int n , a ,b;
    cin >> n >> a >>b;
    string s;
    cin >> s;
    int cnt =0;
    for(int i =0 ;i < s.size();i++){
        if(s[i] > 'A' && s[i] < 'Z'){
            cnt++;
        }
    }
    if(a * cnt > b*(n - cnt)){
        cout << b*(n - cnt) ;
    }else cout << a * cnt;

    return 0;
}